File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let result= []
    for(let i = 0;i<nums.length-1;i++){
        let num1 = nums[i]
        for(let j = i+1; j<nums.length;j++){
            let num2 = nums[j]
            if(num1 + num2 ===target){
                result.push(i)
                result.push(j)
                return result
            }
        }
    }
    
};

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> result;
        for(int i = 0;i<nums.size()-1;i++){
            int num1 = nums[i];
            for(int j = i+1; j<nums.size();j++){
                int num2 = nums[j];
                if(num1 +num2 == target){
                    result.push_back(i);
                    result.push_back(j);
                    return result;
                }
            }
        }
        return result;
    }
};